site/en/docs/extensions/reference/sidePanel/index.md

```diff
@@ -0,0 +1,219 @@
+---
+api: sidePanel
+---
+
+{% Aside 'important' %}
+The Side Panel API will launch in [Chrome 114](https://chromiumdash.appspot.com/schedule), currently in beta and available in stable in late May 2023.
+{% endAside %}
+
+## Overview {: #overview }
+
+Chrome features a built-in side panel that enables users to view more information alongside the main content of a webpage. The Side Panel API allows extensions to display their own UI in the side panel. 
```

Comment:
Would suggest: "... their own UI in the side panel, enabling persistent experiences that complement the user's browsing journey."

Reply:
Love it! Let's do that 👍 💯